src/aspire/utils/filters.py

```diff
@@ -223,7 +223,9 @@ def _evaluate(self, omega):
         return result
 
     def scale(self, c):
-        self._scale *= c
```

Comment:
To make things more consistent with the other filters, let's create a `ScaledFilter` class that wraps the original filter and multiplies the input with a scale. That way we don't have to keep this internal `_scale` variable around everywhere.

Reply:
Done in commit 61bf2d5.